shared/components/menu/menuForm.js

```diff
@@ -0,0 +1,64 @@
+import React from 'react';
+require('es6-promise').polyfill();
+require('isomorphic-fetch');
+
+class MenuForm extends React.Component {
+  constructor(props) {
+      super(props);
+      this.handleRadio = this.handleRadio.bind(this);
+      this.handleTitle = this.handleTitle.bind(this);
+      this.handleSubmit = this.handleSubmit.bind(this);
+      this.state = {
+        chef: 1,
```

Comment:
this is hard coded right now so we're going to have to change it once we figure out this django template stuff. Do you think we should keep it as is for now or wait to merge this frontend stuff when we have it more setup?